code/game/objects/items/weapons/twohanded.dm

```diff
@@ -92,6 +92,13 @@
 	user.put_in_inactive_hand(O)
 	return TRUE
 
+/obj/item/twohanded/mob_can_equip(mob/M, slot) //Unwields twohanded items if they're equipped on the back
+	if(wielded && slot_flags)
+		var/obj/item/twohanded/O = M.get_inactive_hand()
+		if(istype(O))
+			O.unwield(M)
+	return slot_flags
+
```

Comment:
Returning `slot_flags` here is problematic. For example, this makes it possible to equip a fire axe into any open inventory slot (hand slot, pocket slots, etc.). Doing a `return ..()` is probably best here.

Also is there a significance of checking `slot_flags` in the `if`?